sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/src/Internals/Manager.State.cs

```diff
@@ -22,28 +22,40 @@ namespace Azure.Monitor.OpenTelemetry.LiveMetrics.Internals
     /// </remarks>
     internal partial class Manager
     {
-        private Action _callbackAction = () => { };
+        private Thread? _backgroundThread;
```

Comment:
All of these variables have a generic names. Do you want to rename based on its operation?

Reply:
This was the latest rename :D

"backgroundThread" _is_ our background thread. I could rename this to "stateMachineThread". In my opinion, that's not a significant difference.
"shutdownEvent" is primarially used for our thread timer. I'll reflect on if there's a more suitable name for that.

Reply:
Discussed offline. will address refactoring/renaming at a later date.